DOCUMENTATION.md

```diff
@@ -696,9 +696,9 @@ elements you'll touch in a normal application are:
   internal implementation of Optimistic Concurrency Control / retry loop used
   by `Stream`. It's recommended to at least scan this file as it defines the
```

Comment:
You left one here 😉

Reply:
Thanks for reviewing and catching; That was actually only the head of a seam, lots of fixes pushed 😊